chrome/src/js/autocomplete.js

```diff
@@ -0,0 +1,43 @@
+$( function() {
+  let players = getPlayerData()
+
+  const positionMap = [
+    'n/a',
+    'P',
+    'C',
+    '1B',
+    '2B',
+    '3B',
+    'SS',
+    'LF',
+    'CF',
+    'RF',
+    'DH'
+  ]
+
+  $( "#nameInput" ).autocomplete({
+    minLength: 2,
```

Comment:
Is this enough to make sure we're not getting many dupes?

Reply:
thats not what this does. it is how many characters before the autocomplete kicks in